modules/tinymce/src/core/test/ts/browser/init/ShadowDomEditorTest.ts

```diff
@@ -14,8 +14,8 @@ describe('browser.tinymce.core.init.ShadowDomEditorTest', () => {
     }
   });
 
-  const isSkin = (ss: StyleSheet) => ss.href !== null && Strings.contains(ss.href, 'skin.min.css');
-  const isShadowDomSkin = (ss: StyleSheet) => ss.href !== null && Strings.contains(ss.href, 'skin.shadowdom.min.css');
+  const isSkin = (ss: StyleSheet) => ss.href !== null && Strings.contains(ss.href, 'skin.css');
```

Comment:
Wouldn't these need to check both skin.min.css and skin.css not it only matches the latter.

Reply:
in code, yes. In tests there is no suffix unless we set one - that's the new test I added.